### PR TITLE
[WIP][FEAT] - homepage ui

### PR DESCRIPTION
**Pull request by abhij1607**
This PR consist of initial folder structure and components for static homepage UI

### CODE REVIEW

File: src/components/Footer/Footer.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.footer {
+  background-color: var(--dark-color);
```
Comment (hinanshisuthar) [inline src/components/Footer/Footer.css:2]: Good using variables

File: src/styles/base.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
+:root {
```
Comment (hinanshisuthar) [inline src/styles/base.css:1]: Liked how you have initialised variables in start

**Review by hinanshisuthar — COMMENTED** [review]
Clean code practices followed. Minimal UI. Good use of variables. Ready to merge

**Review by stutisk — APPROVED** [review]
good use of variables and ui looks clean

**Pull request merged**